### PR TITLE
New package: python-mechanize-0.2.5

### PR DESCRIPTION
**Pull request by Grauwolf**

### CODE REVIEW

File: srcpkgs/python-mechanize/template [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Template file for 'python-mechanize'
+_name=mechanize
```
Comment (asergi) [inline srcpkgs/python-mechanize/template]: Do not add unnecessary custom variables. Please remove it.

File: srcpkgs/python-mechanize/template [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+pkgname="python-${_name}"
+version=0.2.5
+revision=1
+wrksrc="${_name}-${version}"
+noarch=yes
+build_style=python-module
+hostmakedepends="python python-setuptools"
```
Comment (asergi) [inline srcpkgs/python-mechanize/template]: You can remove python from hostmakedepends.

File: srcpkgs/python-mechanize/template [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+depends="python"
+short_desc="Stateful programmatic web browsing in Python"
+maintainer="Farhad Shahbazi <grauwolf@geekosphere.org>"
+license="BSD"
+homepage="http://wwwsearch.sourceforge.net/${_name}/"
+distfiles="${homepage}/src/${_name}-${version}.tar.gz"
```
Comment (asergi) [inline srcpkgs/python-mechanize/template]: Use distfile from PyPI:
distfiles="${PYPI_SITE}/m/mechanize/mechanize-${version}.tar.gz"

Comment (asergi) [inline srcpkgs/python-mechanize/template]: Add missing pycompile_module line:
pycompile_module="mechanize"

**Comment by Grauwolf** [comment]
@asergi thanks! fixed!

**Pull request merged**